vms/proposervm/vm.go

```diff
@@ -428,72 +438,54 @@ func (vm *VM) LastAccepted(ctx context.Context) (ids.ID, error) {
 	return lastAccepted, err
 }
 
-func (vm *VM) repairAcceptedChainByHeight(ctx context.Context) error {
+func (vm *VM) rollInnerVMAcceptedBlockForward(ctx context.Context) error {
```

Comment:
this can be split out to a separate PR to switch from rolling the outer VM backwards to roll the inner VM forwards